iOS/UI/Swipeable Input/SwipeableInputAccessoryView.swift

```diff
@@ -85,16 +85,17 @@ class SwipeableInputAccessoryView: View, UIGestureRecognizerDelegate {
 
         self.inputContainerView.showShadow(withOffset: 8)
 
-        if let controller = UIWindow.topMostController() {
-            controller.addChild(viewController: self.inputTypeVC, toView: self.inputTypeContainer)
-        }
-
-        self.inputTypeVC.view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Removing this for now because we need to implement the this as an InputVC. If you try to add any controller that exists in here to a non input vc, it will crash.